lib/susedistribution.pm

```diff
@@ -484,8 +484,9 @@ sub init_consoles {
             || is_svirt_except_s390x))
     {
         my $tty = is_agama() ? 8 : 2;
+        my $installation_tty = is_agama() ? 2 : check_var('VIDEOMODE', 'text') ? 1 : 7;
```

Comment:
could you introduce the ternary in the same line instead of declaring new variable, same for `$tty` which also doesn't a proper name. If the line is too long you can break it:
Example 1:
```
$self->add_console('installation', 'tty-console', 
  {tty => check_var('VIDEOMODE', 'text') ? 1 : is_agama() ? 2 : 7});
```
Example 2:
`$self->add_console('install-shell', 'tty-console', {tty => is_agama() ? 8 : 2});`

Additionally when is VIDEOMODE text I guess it should be 1, you should try with more architectures/backends.